File: Python Word examples/mailMerge/mailMerge.py
import spirecloudword
from spirecloudword import MailMergeData
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.mail_merge_api.MailMergeApi(configuration)

file_name = "MailMerge.doc"
merging_list = [MailMergeData('Contact Name', 'E-iceblue'), MailMergeData('Fax', '123456789'),
                MailMergeData('Date', '2019-10-16')]
extra = {'folder': 'input',
         'dest_file_path': 'output/MailMerge.doc'}
api.mail_merge(file_name, merging_list, **extra)

File: JavaScript Word Examples/MailMergeApiDemo/mailMerge.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.MailMergeApi(apiClient);
    
    var name = "MailMergeFormField.docx";
    var destFilePath = 'output/MailMerge/MailMerge.docx';
    var data = [
        new Spirecloudword.MailMergeData("Contact Name", "John Smith"),
        new Spirecloudword.MailMergeData("Fax", "+1 (69) 123456"),
        new Spirecloudword.MailMergeData("Date", "2019-10-16"),
        new Spirecloudword.MailMergeData("Urgent", "Yes"),
        new Spirecloudword.MailMergeData("Share", "No"),
        new Spirecloudword.MailMergeData("Submit", "Yes"),
        new Spirecloudword.MailMergeData("Body", "It's very urgent. Please deal with it ASAP.")
    ];
    var opt = {folder: 'mailmerge'};
    instance.mailMerge(name, destFilePath, data, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();